worblehat-web/src/main/java/de/codecentric/psd/worblehat/web/controller/BorrowBookController.java

```diff
@@ -13,9 +13,11 @@
 import org.springframework.web.bind.annotation.ModelAttribute;
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RequestMethod;
+import org.thymeleaf.util.SetUtils;
```

Comment:
This is the wrong import. You probably meant to import SetUtils from Apache Commons Collections.

Reply:
Used books.isEmpty instead of CollectionUtils.isEmpty(books)